paper.tex

```diff
@@ -457,18 +457,18 @@ \section{Results}
 
 \subsection{CPU: Primitive Count and Timing}
 
-We measured the timing of our CPU implementation, generating both line and arc primitives, against Skia's stroker, which outputs a mix of straight lines and quadratic Bézier primitives. The results are shown in Figure~\ref{fig:timings}. The time shown is the total for test files from the \emph{timings} dataset of \cite{Nehab2020}, and the tolerance in all cases was 0.25. TODO: measure against other implementations.
+We measured the timing of our CPU implementation, generating both line and arc primitives, against the Nehab and Skia strokers, which both output a mix of straight lines and quadratic Bézier primitives. The results are shown in Figure~\ref{fig:timings}. The time shown is the total for test files from the \emph{timings} dataset of \cite{Nehab2020}, and the tolerance was 0.25 when adjustable as a parameter.
 
 Similarly to \cite{Nehab2020}'s observations, Skia is the fastest stroke expansion algorithm we measured. The speed is attained with some compromises, in particular an imprecise error estimation that can sometimes generate outlines exceeding the given tolerance. We confirmed that this behavior is still present. The Nehab paper proposes a more careful error metric, but a significant fraction of the total computation time is dedicated to evaluating it.
 
 \begin{figure}
     \includegraphics[scale=0.6]{timings}
-    \caption{CPU timings for stroke expansion, comparing two versions of our stroker (outputting line and arc primitives) against Skia's curve-to-curve stroker. Measured on an Apple M1 Max}
+    \caption{CPU timings for stroke expansion, comparing two versions of our stroker (outputting line and arc primitives) against Skia's curve-to-curve stroker. Measured on an AMD 3970}
     \Description{A stacked bar chart of the timings for stroke expansion}
     \label{fig:timings}
 \end{figure}
 
-We also compared the number of primitives generated, for varying tolerance values from 1.0 to 0.001. The number of arcs generated is significantly less than the number of line primitives. The number of arcs and quadratic Bézier primitives is comparable at practical tolerances, but the count of quadratic Béziers becomes more favorable at finer tolerances. These vertical axis shows the sum of output segment counts for test files from the \emph{timings} dataset.
+We also compared the number of primitives generated, for varying tolerance values from 1.0 to 0.001. The number of arcs generated is significantly less than the number of line primitives. The number of arcs and quadratic Bézier primitives is comparable at practical tolerances, but the count of quadratic Béziers becomes more favorable at finer tolerances. These vertical axis shows the sum of output segment counts for test files from the \emph{timings} dataset, plus mmark-70k, and omitting the \emph{waves} example\footnote{The \emph{waves} example triggers triggers Skia issue 336617138 at finer tolerance.}.
```

Comment:
`mmark-70k` is defined in the next (GPU) subsection. After we land this and #16 we should probably do another pass and move the stress test scene definitions to the "Results" section introduction (right before this subsection).